app/src/main/java/org/gdg/frisbee/android/adapter/DrawerAdapter.java

```diff
@@ -40,8 +40,9 @@ private void initAdapter() {
             }
             add(new DrawerItem(Const.DRAWER_ACHIEVEMENTS, R.drawable.ic_drawer_achievements, R.string.achievements));
             add(new DrawerItem(Const.DRAWER_ARROW, R.drawable.ic_drawer_arrow, R.string.arrow));
-            add(new DrawerItem(Const.DRAWER_SETTINGS, R.drawable.ic_drawer_settings, R.string.settings));
-            add(new DrawerItem(Const.DRAWER_ABOUT, R.drawable.ic_drawer_about, R.string.about));
+            add(new DrawerItem(Const.DRAWER_SETTINGS, android.R.drawable.ic_menu_preferences, R.string.settings));

```

Comment:
Are you sure about this? android.R version of drawables are changing very frequenlty and different in all platform versions